### PR TITLE
Avoid use C-style array declaration.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java [hunk 1]
```diff
@@ -244,7 +244,7 @@ public String[] parseReferences(final String name, final InputStream in, final B
      * @throws Pack200Exception if a problem occurs with an unexpected value or unsupported Codec
      */
     public String[][] parseReferences(final String name, final InputStream in, final BHSDCodec codec,
-        final int counts[], final String[] reference) throws IOException, Pack200Exception {
+                                      final int[] counts, final String[] reference) throws IOException, Pack200Exception {
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java:247]: Leave this formatting alone, please; it creates too much noise to review in the PR.

File: src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java [hunk 2]
```diff
@@ -172,8 +172,8 @@ public long[] parseFlags(final String name, final InputStream in, final int coun
         return parseFlags(name, in, new int[] {count}, hiCodec, loCodec)[0];
     }
 
-    public long[][] parseFlags(final String name, final InputStream in, final int counts[], final BHSDCodec hiCodec,
-        final BHSDCodec loCodec) throws IOException, Pack200Exception {
+    public long[][] parseFlags(final String name, final InputStream in, final int[] counts, final BHSDCodec hiCodec,
+                               final BHSDCodec loCodec) throws IOException, Pack200Exception {
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java:176]: Leave this formatting alone, please; it creates too much noise to review in the PR.

File: src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java [hunk 3]
```diff
@@ -162,8 +162,8 @@ public long[] parseFlags(final String name, final InputStream in, final int coun
         return parseFlags(name, in, new int[] {count}, (hasHi ? codec : null), codec)[0];
     }
 
-    public long[][] parseFlags(final String name, final InputStream in, final int counts[], final BHSDCodec codec,
-        final boolean hasHi) throws IOException, Pack200Exception {
+    public long[][] parseFlags(final String name, final InputStream in, final int[] counts, final BHSDCodec codec,
+                               final boolean hasHi) throws IOException, Pack200Exception {
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java:166]: Leave this formatting alone, please; it creates too much noise to review in the PR.

**Review by garydgregory — CHANGES_REQUESTED** [review]
Leave this formatting alone, please; it creates too much noise to review in the PR.

File: src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java (hunk 3 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java]: Done

File: src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java (hunk 2 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java]: Done

File: src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java (hunk 1 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java]: Done

**Pull request merged**